### ISSUE TITLE
Jerry forgets "Maximize" once reopened

### ISSUE THREAD

**Issue by futurelauncher** [asdfjkl/jerry#134]
Hi,
If you maximize the window and close it. Later when you open it again, you see the window in "Restore Down" position. It should be maximized only, obviously. Why one should take the efforts to maximize it every time we open it?

**Comment by asdfjkl** [comment]
I cannot reproduce this behavior. Are you talking about maximizing the program windows (which should be remembered) or about full-screen mode? Which operating system are you using?

**Comment by futurelauncher** [comment]
Yes, I am talking about maximizing the program window (which should be remembered). I am on Windows 10.

**Comment by asdfjkl** [comment]
at last! finally fixed in fbd22e5
